package.json

```diff
@@ -11,29 +11,37 @@
   "bugs": {
     "url": "https://github.com/MAARK/code-config/issues"
   },
+  "bin": {
+    "howl": "lib/index.js"
+  },
   "scripts": {
-    "test": "concurrently npm:test:*",
-    "fix": "concurrently 'npm:test:* -- {1}' -P -- --fix",
-    "test:eslint": "node ./test/eslint.js",
-    "test:stylelint": "node ./test/stylelint.js"
+    "start": "npm run build && node lib/index.js",
+    "build": "esbuild src/index.js --bundle --minify --platform=node --outfile=lib/index.js --target=node12",
+    "watch": "npm run build -- --watch",
+    "lint": "eslint '**/*.js'",
+    "test": "concurrently -c 'cyan,yellow,green' npm:test:*",
+    "fix": "concurrently -c 'cyan,yellow,green' 'npm:test:* -- {1}' -P -- --fix",
+    "test:eslint": "node ./test/selfeslint.js",
+    "test:stylelint": "node ./test/selfstylelint.js"
   },
-  "peerDependencies": {
-    "eslint": "^8.18.0",
+  "devDependencies": {
+    "concurrently": "^7.2.2",
+    "esbuild": "^0.17.11",
+    "eslint": "^8.35.0",
     "eslint-config-airbnb": "^19.0.4",
-    "eslint-config-prettier": "^8.5.0",
+    "eslint-config-prettier": "^8.7.0",
+    "eslint-import-resolver-alias": "^1.1.2",
     "eslint-plugin-prettier": "^4.2.1",
-    "prettier": "^2.7.1",
-    "stylelint": "^14.9.1",
-    "stylelint-config-standard": "^26.0.0",
-    "stylelint-config-standard-scss": "^4.0.0"
+    "prettier": "^2.8.4",
+    "stylelint": "^15.2.0",
+    "stylelint-config-standard": "^30.0.1",
+    "stylelint-config-standard-scss": "^7.0.1"
   },
   "dependencies": {
+    "enquirer": "^2.3.6",
     "prop-types": "^15.8.1",
     "react": "^18.2.0"
```

Comment:
I'm missing one commit to remove these two dependencies because I would rather mock them to avoid downloading unnecessary libraries. I can create a different PR for this or just add it here.